source/utils/types.ts

```diff
@@ -116,15 +114,10 @@ export interface Delays {
 export type Headers = Record<string, string | string[]>;
 
 // The library overrides the type definition of `agent`, `host`, 'headers and `timeout`.
-export interface Options extends Except<https.RequestOptions, 'agent' | 'timeout' | 'host' | 'headers'> {
-	host?: string;
+export interface Options extends URLOptions {
```

Comment:
TODO: Lots of HTTP options missing here.

Reply:
Add a TODO comment in the code if you intend to fix this later.